index.html

```diff
@@ -0,0 +1,174 @@
+                        Wrestler, a dream coming true and an unforgetable moment</p></div>
+                </div></div>
+            </div>
+            <h3 class="text-center pt-8 hidden md:block text-2xl"><a href="#" class="underline  program-link">View full
+                    Program</a>
+            </h3>
+            <div class="py-6 px-12 join-btn md:hidden mt-8" id="invite">
+                Join The WWE International Rendezvous 2023
+            </div>
+        </section>
+
+        <section class="py-12 px-[20px] flex flex-col items-center w-full guests" id="guests">
+            <h2 class="text-center text-3xl guests-title pt-8">Featured Guests</h2>
+            <div class="line"></div>
+            <div class="mt-12 md:grid grid-cols-2 md:gap-8 mx-3 md:mx-10 guests-container" id="guests-container">
+            </div>
+            <div class="py-6 px-12 join-btn md:hidden mt-8" id="more" onclick="unhidden()">
+                More
+            </div>
+        </section>
+
+        <section class="partners py-12" id="partners">
+            <h2 class="text-center text-3xl partners-title pt-8">Partners</h2>
+            <div class="line"></div>
+            <div class="w-full py-5 mt-5">
+                <div class="w-11/12 mx-auto">
+                    <div class="grid grid-cols-3 md:grid-cols-6 gap-4  justify-between ">
+                        <div class="partnerImg">
+                            <img src="img/monster.png" class="partnerLogo" alt="Responsive image">
+                        </div>
+                        <div class="partnerImg">
+                            <img src="img/corona.png" class="partnerLogo" alt="Responsive image">
+                        </div>
+                        <div class="partnerImg">
+                            <img src="img/pretorian.png" class="partnerLogo" alt="Responsive image">
+                        </div>
+                        <div class="partnerImg">
+                            <img src="img/snickers.png" class="partnerLogo" alt="Responsive image">
+                        </div>
+                        <div class="partnerImg">
+                            <img src="img/Optimum_Nutrition.png" class="partnerLogo" alt="Responsive image">
+                        </div>
+                        <div class="partnerImg">
+                            <img src="img/wells-fargo.png" class="partnerLogo" alt="Responsive image">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+    </main>
+    <footer class="footer bottom-0 w-full py-8 flex items-center justify-center bg-[var(--font-second)]">
+
+        <h3 class="text-xl md:text-2xl font-bold px-4 md:pr-16 footer-title text-black">WWE International Rendezvous</h3>
+        <div>
+            <p class="text-black">Original Design by Cindy Shin in Behance</p>
+            <p class="text-black">Some Rights Reserved</p>
+        </div>
+    </footer>
```

Comment:
- [ OPTIONAL ] Great job so far but I think it would be great if you remove the footer from the home page in the mobile version since there is no footer in the mobile version home page in the given design so I highly recomend to reomve it from there just from the mobile version ✅💯🚀 I am assigning it as optional because the previous reviewer didn't mention it but it still important to fix it 💯🔥🔥💪🏻


- Your
![image](https://user-images.githubusercontent.com/58553711/233234767-a068ad89-53ba-4d48-9c8d-51da6676b910.png)


- Given
![image](https://user-images.githubusercontent.com/58553711/233234894-c0d79784-6178-4b0d-baf6-a327a20e2c46.png)